dashboard/config/routes.rb

```diff
@@ -82,7 +82,8 @@
 
     resources :images, only: [:new]
 
-    get "/ai_tutor/tester", to: "ai_tutor#tester"
+    get "/ai_iteration/tools", to: "ai_iteration#tools"
```

Comment:
I would suggest using ability.rb to block this page for users who are not at least levelbuilders. 

please also add ruby unit tests for your endpoints validating that you get the correct response code for signed out user, non-levelbuilder user, levelbuilder user, and levelbuilder with tutor access. these are quick to write with `test_user_gets_response_for`